packages/compiler-sfc/__tests__/__snapshots__/compileScript.spec.ts.snap

```diff
@@ -963,12 +966,12 @@ import { ref } from 'vue'
         import Foo, { bar } from './Foo.vue'
         import other from './util'
         import * as tree from './tree'
+        const constant = {}
```

Comment:
shouldn't objects (and non literals) stay inside `setup()`? Since they can be modified, it makes sense to have one per component

Reply:
I don't think so, since each SFC can only have a component.

Could you please provide a case that you described?

Reply:
Sure, [here](https://sfc.vuejs.org/#eNp9UctOwzAQ/JWVL0lVanPhUqUIBBcuIHH2JXU3kBI/ZDupUJp/Z51HxaFw213P7Myse/boHO9aZFtWBOVrFyFgbN29NDLW2lkf4clqB5W3GjIuUpMIGQFMISZOQpsionZNGZE6gGJkdZvK+p1kNdQG7iQDQY+FuCDZDZtENrp0/BisISN94hNnfAiSbWGcpBkJp16yzxhd2AoRKpXcHAO3/kNQxX1ryDhyDHqz9/YU0NNiydKKQZqBJJcMV0LPkXvwWMEwp17iKmtCBP39as07lirWHb7tj6gi7KCn3QvCUE/8/HaVZlVrCGoNnUB51GhivpoSGd6VTYvrdWqurOUHuhHtMniCZypz2kcy/139UHdjQeW+jZFUH1RTq6/0B4u8ZDME4OUymzhiIs3vTUlhLjTI+56iDcMKSrpQ/7fj8xkygx36jNCjLTH5+v3zww8fqdtN)

if you host that literal object, they will share the date.
Symbols also need to be handled: they whole point is for them to be unique so if they are created within the `setup()` function they should stay there and be unique to each component instance

Reply:
I see, thanks! I will fix it later.